_includes/pbc/all/install-features/202410.0/install-dynamic-multistore.md

```diff
@@ -1544,3 +1544,135 @@ docker/sdk boot && docker/sdk up --assets
+|StorePrefixStorageRouterEnhancerPlugin| Extracts and adding the current store name to the Route parameters, and for prefixing the generated URL with the current store name. Through the \SprykerShop\Yves\StorageRouter\StorageRouterConfig::getAllowedStores() configuration, you tell this plugin what store names it has to work with. ||SprykerShop\Yves\StorageRouter\Plugin\RouterEnhancer|
+|GeneratePathTwigPlugin| Wrapper on Symfony Router `generate()` function.                                                                                                                                                                                                                                                   ||SprykerShop\Yves\ShopUi\Plugin\Twig|
+
+
+
+{% info_block warningBox "Verification" %}
+
+In the RouterDependencyProvider::getRouterEnhancerPlugins() stack, the order of plugin execution has changed:
+•   By default store name placed before the language prefix in the URL. Example: `https://yves.eu.mysprykershop.com/DE/en/`.    
+•	So the `StorePrefixRouterEnhancerPlugin` must now be executed before the `LanguagePrefixRouterEnhancerPlugin`.
+•	Ensure your plugin stack reflects this updated order to maintain the correct routing behavior.
+
+{% endinfo_block %}
+
+**src/Pyz/Yves/Router/RouterDependencyProvider.php**
+
+```php
+<?php
+namespace Pyz\Yves\Router;
+
+use Spryker\Yves\Router\Plugin\RouterEnhancer\LanguagePrefixRouterEnhancerPlugin;
+use Spryker\Yves\Router\Plugin\RouterEnhancer\StorePrefixRouterEnhancerPlugin;
+use Spryker\Yves\Router\RouterDependencyProvider as SprykerRouterDependencyProvider;
+
+
+class RouterDependencyProvider extends SprykerRouterDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Yves\RouterExtension\Dependency\Plugin\RouterEnhancerPluginInterface>
+     */
+    protected function getRouterEnhancerPlugins(): array
+    {
+        return [
+            new StorePrefixRouterEnhancerPlugin(),
+            new LanguagePrefixRouterEnhancerPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Yves/StorageRouter/StorageRouterDependencyProvider.php**
+
+```php
+<?php
+namespace Pyz\Yves\StorageRouter;
+
+use SprykerShop\Yves\StorageRouter\Plugin\RouterEnhancer\StorePrefixStorageRouterEnhancerPlugin;
+use SprykerShop\Yves\StorageRouter\StorageRouterDependencyProvider as SprykerShopStorageRouterDependencyProvider;
+
+
+class StorageRouterDependencyProvider extends SprykerShopStorageRouterDependencyProvider
+{
+    /**
+     * @return array<\SprykerShop\Yves\StorageRouterExtension\Dependency\Plugin\StorageRouterEnhancerPluginInterface>
+     */
+    protected function getStorageRouterEnhancerPlugins(): array
+    {
+        return [
+            new StorePrefixStorageRouterEnhancerPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+We do not need `LanguagePrefixRouterEnhancerPlugin` for the StorageRouter, as the language is already part of the URL for the StorageRouter.
```

Comment:
What do I need to do with this information?

Reply:
just for information) 
it is request from Serhii https://github.com/spryker/spryker-docs/pull/2940#discussion_r1875827906